tests/integration_tests/document_loaders/test_mongodb.py

```diff
@@ -1,17 +1,47 @@
+import os
+
+import pytest
+
 from langchain.document_loaders.mongodb import MongodbLoader
 
+if "MONGODB_API_KEY" in os.environ:
+    mongo_api_key_set = True
+    mongo_api_key = os.environ["MONGODB_API_KEY"]
+else:
+    mongo_api_key_set = False
 
-def test_load() -> None:
-    connection_string = "mongodb://localhost:27017/"
-    db_name = "sample_restaurants"
-    collection_name = "restaurants"
+
+@pytest.fixture
+def mongodb_loader():
+    connection_string = "mongodb://localhost:27017"
```

Comment:
this can be a challenge to auto tests. During CI process, each time a new "virtual machine" is created with our predefined env setup and dependencies before running our auto tests here. The assumption here is that there is a local mongoDB host service (and several env variables), which is not true in auto test environment.

to make it work, 
- we will need to provide the whole setup in github CI workflow config, the dependencies will need to be installed too. (interesting, I can only see unit_test here. @vowelparrot / @dev2049 , do you know where are integration_tests configured? Thanks) (unittest config: https://github.com/hwchase17/langchain/blob/master/.github/workflows/test.yml)
- alternatively, we can make a mock mongodb API with predefined answer for each specific request.